test-cmdline.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
It looks like this just exercises OpenSSL, not the code in this module?  I'm confused what the purpose is..

Reply:
Same here...

Reply:
the purpose is to show that files which validate using the OpenSSL binary also validate in this library.  If the library test fails, we know whether it's because the test-files are valid or not based on whether this test works or not.

Reply:
It might be good to add some comments to that effect since at least two of us were confused.  I was particularly confused since this seems to benchmark openssl and run it under valgrind.  I can see the use of benchmarking in development, but since none of the code in this repo is involved, I don't understand the use of valgrind.

Perhaps make benchmarking and valgrind optional, so that running the tests doesn't wait for it?